docs/book/src/reference/creating-events.md

```diff
@@ -0,0 +1,109 @@
+# Creating Events
+
+It is often useful to publish *Event* objects from the controller Reconcile function. 
+
+Events allow users to see what is going on with a particular object, and allow automated processes to see and respond to them.
+
+ Recent Events for an object can be viewed by running `kubectl describe <resource kind> <resource name>`.
+ 
+  ### how to implement the solution to raise an event:
```

Comment:
```suggestion
## How to raise events? 

Following are the steps with examples to help you raise events in your controller's reconciliations. 
```

Reply:
Let's add the definition with an event within its usage example.
And then, afterwords, provide a detailed example of how to implement it fully. 

It seems more natural. You will first read what it is, how to use it, and its purpose. You only go to the details when/if you need to.